changelog.d/7864.bugfix

```diff
@@ -0,0 +1 @@
+Fix in-memory federation transmission queues building up indefinitely.
```

Comment:
What's this mean to an end-user? Is there a behavior that is fixed with this?

Reply:
How about

> Stop leaking memory when remote servers have been unreachable for a long time.

?

Reply:
Maybe:

> Fix a memory leak by limiting the number of messages queued for a remote server that has been unreachable for a long time.